require_test/example.js

```diff
@@ -0,0 +1,11 @@
+// Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
```

Comment:
Move that directory to `cli/tests/testdata/require` or `cli/tests/testdata/commonjs`